.github/workflows/new-app-review/validators/package-json.ts

```diff
@@ -0,0 +1,34 @@
+import { hasPackageJson, packageJsonPath } from '../../app-review-utils';
+import type { PullRequestFile, ValidatorOptions, ValidatorResult } from '../../types';
+
+export const validate = async (_options: ValidatorOptions, newAppDir: string, files: PullRequestFile[]): Promise<ValidatorResult> => {
```

Comment:
one thing we had to do recently was validate that there was NOT a deploy script, is that worth adding here as well?

Reply:
🙏 that script is just called `deploy`, yeah? 2f89a41d7882fa168a4dca617d6ddd24a99ff778